Cabal/src/Distribution/PackageDescription/Check.hs

```diff
@@ -669,6 +670,12 @@ ppExplanation (UnknownArch unknownArches) =
     "Unknown architecture name " ++ commaSep (map quote unknownArches)
 ppExplanation (UnknownCompiler unknownImpls) =
     "Unknown compiler name " ++ commaSep (map quote unknownImpls)
+ppExplanation (MissingUpperBounds names) =
+    "These packages miss upper bounds '"
+      ++ (intercalate "','" (unPackageName <$> names)) ++ "'"
+      ++ " please add them with with `cabal gen-bounds`."
```

Comment:
"with with" --> "e.g. using"
`cabal gen-bounds` cannot add these bounds directly, but can print out suggestions.

Reply:
I changed that into "please add them using `cabal gen-bounds` for suggestions."